qxf2/newsletter_automation
# Add Article form: Added wtforms with time component on TimeField

This PR is created to solve the issue-
https://trello.com/c/uyMXFHz6/41-add-articles-has-no-validation-for-time-field

I have added wtforms component with TimeFiled for Add Article Form.

To test this:
1. Launch newsletter app locally.
2. Add time in minutes.
3. Check the form in Manage article. It will show time added as 08:00, 16:00 Min etc.
